frontend/src/components/AgencyProfile/AgencySideBar.jsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import { Component, useEffect, useState } from "react";
+import "./AgencyProfile.css";
+import { NavHashLink } from "react-router-hash-link";
+
+/**
+ * Functional component for Agency Side Bar
+ *
+ * @param {*} getScrollPositions function that returns page offset of agency categories
+ * @param {string} id unique id of agency
+ * @param {*} ScrollTo function that scrolls page to given element
+ * @returns {*} Agency Side Bar Component
+ */
+function AgencySideBar({ getScrollPositions, id, positions, ScrollTo }) {
```

Comment:
Neither `id` nor `positions` is used - remove; also consider renaming `ScrollTo` to `scrollTo`